src/test/java/org/folio/circulation/domain/policy/PeriodTest.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.circulation.domain.policy;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+@RunWith(JUnitParamsRunner.class)
+public class PeriodTest {
+
+  @Test
+  @Parameters({
+    "Minutes | 6  | 6",
+    "Hours   | 5  | 300",
+    "Days    | 4  | 5760",
+    "Weeks   | 3  | 30240",
+    "Months  | 2  | 89280",
+    "Random  | 9  | 9"
```

Comment:
What does the `Random` option represent? Is that mean to reflect an interval the system does not recognise?

Reply:
Yes.

Reply:
I think that is a separate test, as it describes what to do if the period is invalid (which really shouldn't be allowed).

Do we want the behaviour of an unrecognised interval to assume that it should be treated as minutes, rather than fail?

Reply:
> Do we want the behaviour of an unrecognised interval to assume that it should be treated as minutes, rather than fail?

It does not make much sense to me, but that is what the requirements say:
```
    If Grace period interval = Hour(s)
        Set minutesGracePeriod = Grace period * 60
    Else if Grace period interval = Day(s)
        Set minutesGracePeriod = Grace period * 1440
    Else if Grace period interval = Week(s)
        Set minutesGracePeriod = Grace period * 10080
    Else if Grace period interval = Month(s)
        Set minutesGracePeriod = Grace period * 44640
    Else
        Set minutesGracePeriod = Grace period <=====means the grace period is already in minutes
```

Reply:
I think that requirement needs to be explored with the PO. 

I think that else is probably assuming that the last remaining interval is minutes, and isn't considering the invalid interval situation.

Reply:
Thank you for extracting the scenario to a separate test

Reply:
> I think that requirement needs to be explored with the PO.

I agree.

Reply:
Should we wait until the PO has answered your question before merging this pull request?

Reply:
Yes, I think we should wait.

Reply:
I talked to Holly Mistlebauer:
> If you need to do something, then set the grace period to 0 minutes if any other values are encountered.

Reply:
Ok. I would personally prefer a failure, however let's go with Holly's decision